### PR TITLE
For module: add function to get configuration from .yo-rc.json

### PR DESCRIPTION
**Pull request by pascalgrimaud**
Related to this comment https://github.com/jhipster/generator-jhipster-module/commit/a6efe65f12bb221dbddc0e5d5c489686573c2a52#commitcomment-22750182

- Please make sure the below checklist is followed for Pull Requests.

- [ ] [Travis tests](https://travis-ci.org/jhipster/generator-jhipster/pull_requests) are green
- [ ] Tests are added where necessary
- [ ] Coding Rules & Commit Guidelines as per our [CONTRIBUTING.md document](https://github.com/jhipster/generator-jhipster/blob/master/CONTRIBUTING.md) are followed

### CODE REVIEW

**Comment by cbornet** [comment]
I think it's already available in jhipsterVar.jhipsterConfig

**Comment by pascalgrimaud** [comment]
@cbornet : yes but as discussed in our mailing list, we will deprecate all jhipsterVar and jhipsterFunc, and use directly all functions in generator-base file

I have issue with eslint-> I will update my PR

File: generators/generator-base.js [hunk 1]
```diff
@@ -49,6 +49,21 @@ const SERVER_MAIN_RES_DIR = constants.SERVER_MAIN_RES_DIR;
  * The method signatures in public API should not be changed without a major version change
  */
 module.exports = class extends PrivateBase {
+
+    /**
+     * Get the JHipster configuration from your .yo-rc.json
+     */
+     getJhipsterAppConfig() {
```
Comment (deepu105) [inline generators/generator-base.js]: can we add a namespace and default it like below to make the method more generic
```
getJhipsterAppConfig(namespace = 'generator-jhipster') {
  const fromPath = '.yo-rc.json';
  if (shelljs.test('-f', fromPath)) {
              const fileData = this.fs.readJSON(fromPath);
              if (fileData && fileData[namespace]) {
                  return fileData[namespace];
              }
          }
          return false;
      }

File: generators/generator-base.js [hunk 2]
```diff
@@ -50,6 +50,20 @@ const SERVER_MAIN_RES_DIR = constants.SERVER_MAIN_RES_DIR;
  */
 module.exports = class extends PrivateBase {
     /**
+     * Get the JHipster configuration from your .yo-rc.json
+     */
```
Comment (deepu105) [inline generators/generator-base.js]: You missed to add @param doc here for namespace

Comment (pascalgrimaud) [inline generators/generator-base.js]: yes, updated

**Pull request merged**